scripts/make-bundle

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
It wouldn't be very hard to add a small version check at the top and bail out early, I can do that np.